doc/sphinx/source/gettingstarted.md

```diff
@@ -0,0 +1,244 @@
+# Getting Started
+
+## Building
+
+The CEED library, `libceed`, is a C99 library (with Fortran and Python interfaces, see `interface/ceed-fortran.c` and `interface/ceed-python/`, respectively) with no external dependencies. It
```

Comment:
What about:

"The CEED library, `libceed`, is a C99 library with no external dependencies. The library has Fortran and Python interfaces; see `interface/ceed-fortran.c` and `interface/ceed-python/`."

It's a matter of personal taste, but I try to avoid long parentheticals.

If we direct users to look into `interface/ceed-fortran.c`, we might want to tidy up that file?

Reply:
I don't think we should direct them to that file. Direct them to an example (or test) instead.

Reply:
It might be worth it to make a simplified ex1 in Fortran and Python in that case. Then users can look at those as an example.

Reply:
Agreed, but I think that's out of scope for this PR.

Reply:
Yes. I'll make an issue.

Reply:
Updated as suggested. Thanks.